src/site/layouts/home.drupal.liquid

```diff
@@ -0,0 +1,139 @@
+            {% endfor %}
+          </ul>
+        </div>
+        {% if forloop.index == 2 %}
+        </div>
+        <div class="hub-links-row">
+        {% endif %}
+      {% endfor %}
+      </div>
+    </div>
+
+    {% include "src/site/includes/veteran-banner.html" %}
+
+  </section>
+  <!-- /the hub -->
+
+  <section id="content">
+
+    <section id="homepage-benefits">
+      <div class="usa-grid usa-grid-full homepage-benefits-row">
+      {% for hub in hubs %}
+        <div class="usa-width-one-third">
+          <h3 class="heading-level-4"><a href="{{ hub.entity.entityUrl.path }}" onclick="recordEvent({ event: 'nav-linkslist' });"><i class="icon-small icon-heading hub-icon-{{ hub.entity.fieldTitleIcon }} hub-background-{{ hub.entity.fieldTitleIcon }} white"></i>{{ hub.entity.entityLabel }}</a></h4>
+          <p class="homepage-benefits-description">{{ hub.entity.fieldTeaserText }}</p>
+        </div>
+        {% if hub.end_row == true and forloop.last != true %}
+        </div>
+        <div class="usa-grid usa-grid-full homepage-benefits-row">
+        {% endif %}
+      {% endfor %}
+      </div>
+    </section>
+
+    <section id="homepage-popular">
+      <div class="usa-grid usa-grid-full">
+        <div class="usa-width-one-third">
+          <a href="/find-locations/" onclick="recordEvent({ event: 'nav-main-health' });" class="homepage-button">
+            <div class="icon-wrapper">
+              <i class="fa fa-map-marker-alt homepage-button-icon"></i>
+            </div>
+            <!-- div required for alignment -->
+            <div class="button-inner">
+              <span>Find a VA health facility, benefit office, or cemetery</span>
+            </div>
+          </a>
+        </div>
+
+        <div class="usa-width-one-third">
+          <button onclick="recordEvent({ event: 'nav-main-vcl' });" class="homepage-button vcl va-overlay-trigger" data-show="#modal-crisisline">
+            <div class="icon-wrapper vcl"></div>
+            <div class="button-inner">
+              <span>Talk to a Veterans Crisis Line responder now</span>
+            </div>
+          </button>
+        </div>
+
+        <div class="usa-width-one-third" id="myva-login">
+          <button onclick="recordEvent({ event: 'nav-main-sign-in' });" class="homepage-button signin-signup-modal-trigger">
+            <div class="icon-wrapper">
+              <i class="fas fa-user-circle homepage-button-icon"></i>
+            </div>
+            <div class="button-inner">
+              <span>Sign in or create an account to use more tools</span>
+            </div>
+          </button>
+        </div>
+
+      </div>
+    </section>
+
+    <!-- end triple column -->
+
+  </div> <!-- end #content -->
+
+  <section class="usa-grid usa-grid-full">
+    <div class="va-h-ruled--stars"></div>
+  </section>
+
+
+  <section id="homepage-news">
+
+    <div class="usa-grid usa-grid-full">
+    {% for promo in promos %}
```

Comment:
Where does `promos` come from and how does it get here? (Gotta love globally scoped variables, eh? :sweat_smile: Thanks, Liquid! )